superset-frontend/src/explore/components/controls/VizTypeControl/index.tsx

```diff
@@ -46,6 +48,13 @@ const bootstrapData = getBootstrapData();
 const denyList: string[] = bootstrapData.common.conf.VIZ_TYPE_DENYLIST || [];
 const metadataRegistry = getChartMetadataRegistry();
 
+if (
+  isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS) &&
+  !('filter_box' in denyList)
```

Comment:
`in` operator doesn't work as expected for array. Please use `includes` instead

```suggestion
  !denyList.includes('filter_box')
```

Reply:
True @justinpark! That's the problem of switching languages all the time 😄 

@john-bodley You should also use includes in `ChartCreation`